sled-diagnostics/src/queries.rs

```diff
@@ -263,6 +266,99 @@ pub fn pfiles_process(pid: i32) -> Command {
     cmd
 }
 
+pub fn uptime() -> Command {
+    let mut cmd = std::process::Command::new(UPTIME);
+    cmd.env_clear();
+    cmd
+}
+
+pub fn kstat_low_page() -> Command {
+    let mut cmd = std::process::Command::new(PFEXEC);
+    cmd.env_clear().arg(KSTAT).arg("-p").arg("unix::system_pages:low_mem_scan");
+    cmd
+}
+
+pub fn svcs_show_disabled() -> Command {
```

Comment:
Hmm can we rename this to something like `svcs_enabled_but_not_running`

```
       -x
                           Displays explanations for service states.

                           Without arguments, the -x option explains the
                           states of services which:

                               o      are enabled, but are not running.

                               o      are preventing another enabled service
                                      from running.
```

Reply:
Good catch, updated!